src/main/java/net/minecraftforge/common/ToolActions.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2021.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.common;
+
+import net.minecraft.tags.Tag;
+
+import java.util.List;
+
+public class ToolActions
+{
+    /**
+     *  Queried when an axe wants to strip a log
+     */
+    public static final String STRIP = "strip";
+
+    /**
+     *  Queried when an axe wants to scrape copper
+     */
+    public static final String SCRAPE = "scrape";
+
+    /**
+     *  Queried when an axe wants to remove wax out of copper
+     */
+    public static final String WAX_OFF = "wax_off";
+
+    /**
+     *  Queried when a shovel wants to turn dirt into path
+     */
+    public static final String PATH = "path";
+
+    /**
+     *  Queried when a shovel wants to turn dirt into path
```

Comment:
Copy paste error in this javadoc?

Reply:
Yes, although that action isn't implemented so it probably should be removed instead.